CHANGELOG.md

```diff
@@ -7,6 +7,188 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [16.0.0] - 2023-08-XX
+
+### Breaking changes
+
+-   Now only supporting CDI 4.0
+-   Now supporting FDI 1.18
+-   removed the recipe specific `User` type, now all functions are using the new generic `User` type
+-   The `fetchValue` callback of claims now take a new `recipeUserId` param
+-   Now ignoring protected props in the payload in `createNewSession` and `createNewSessionWithoutRequestResponse`
+-   `createdNewUser` has been renamed to `createdNewRecipeUser` in sign up related APIs and functions
+
+-   EmailPassword:
+    -   removed `getUserById`, `getUserByEmail`
+    -   added `consumePasswordResetToken`
```

Comment:
```suggestion
    -   added `consumePasswordResetToken`. This function allows the consumption of the reset password token without changing the password. It will return OK if the token was valid.
```